lib/tasks/resumis.thor

```diff
@@ -0,0 +1,37 @@
+require "thor"
+
+ENV['RAILS_ENV'] ||= 'development'
+require File.expand_path('config/environment.rb')
+
+class Resumis < Thor
+  desc "useradd", "creates a new user"
+  def useradd
+    say("Let's create your user")
+    email = ask('Email: ')
+    password = ask('Password: ', :echo => false)
+    #Add an empty line
+    say('')
+    first_name = check_value('First Name: ')
+    last_name = check_value('Last Name: ')
+    domain = ask('Domain: ')
```

Comment:
`hostname` and `domain` asks should probably be wrapped in a `ResumisConfig.mulit_tenant?` check, since those two fields are relevant only within that context.